pixeltable/functions/fireworks.py

```diff
@@ -29,6 +29,7 @@ def chat_completions(
     top_p: Optional[float] = None,
     temperature: Optional[float] = None,
 ) -> dict:
+    """Pixeltable UDF for the Fireworks `chat/completions` API endpoint."""
```

Comment:
No need to preface with 'Pixeltable UDF for ...', they all are.

(The docstring for str.capitalize() doesn't start with "Python function for ...")